engines/scumm/smush/smush_player.cpp

```diff
@@ -641,7 +641,7 @@ void SmushPlayer::handleTextResource(uint32 subType, int32 subSize, Common::Seek
 	// bit 7 - skip ^ codes (COMI)     0x80        (should be irrelevant for Smush, we strip these commands anyway)
 	// bit 8 - no vertical fix (COMI)  0x100       (COMI handles this in the printing method, but I haven't seen a case where it is used)
 
-	if (flags & 4) {
+	if (flags & 4 || _vm->_language == Common::HE_ISR) {
```

Comment:
Please use parentheses for brevity. Yes, the bitwise operator takes precedence over the logical operator, but parentheses make this clearer

```suggestion
	if ((flags & 4) || _vm->_language == Common::HE_ISR) {
```

Reply:
Thank you, fixed